middleware/error_handler.js

```diff
@@ -0,0 +1,6 @@
+import logger from '../config/logger/logger-service.js';
+
+export default (err, req, res, next) => {
+    res.status(500).send('Oops!');
```

Comment:
what will get the client if the error happened somewhere was 403/401?

Reply:
Added check if error had a previously set status